trap_rain_water_lc42.py

```diff
@@ -0,0 +1,30 @@
+"""
+Method 1 - Brute force -For every element, find the max on either side, thats how much
+water can be stored from that point. In order for water to hold in the crevice, take minimum of
+the height and subtract with currect height
+
+Did it run on LC: No - TLE
+
+Time Complexity: O(n^2) - for every element, you will find max on either side which means n^2 comprision
+
+Space Complexity: O(1) - extra space for variables
+

```

Comment:
Try the 2 pointer approach with the left and rightwall and process the problem

Reply:
Sure thanks Lohit, I will try it out and update my PR